gateway/src/apicast/policy/caching/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Caching 3scale backend response",
+  "description": [
+    "Configures a cache for the authentication calls against the 3scale backend.",
+    "The 3scale backend can authorize (status code = 200) and deny (status code = 4xx) calls.",
+    "When it fails, it returns a 5xx code.."
+  ],
+  "version": "1.1",
+  "configuration": {
+    "$ref": "schema.json#"
```

Comment:
@davidor I'd like to hear what is your though on this. 

This would allow us to have schema and manifest in different JSON files, but we would need to use some tool (like [json-refs](https://github.com/whitlockjc/json-refs)) to merge the doc. It is not a big deal, just pass  it through, I verified it works and creates correct output.

Or we can have the schema embedded in the manifest and have just one, bigger JSON.